src/features/onboarding/welcome/Welcome.tsx

```diff
@@ -1,42 +1,56 @@
 import * as React from 'react'
 import { Link } from 'react-router-dom'
+import cn from 'classnames'
 
-import Typography from '../../../common/components/Typography/Typography'
 import playIco from '../../../common/assets/icons/ico-arrow.svg'
 import image from '../../../common/assets/images/video-placeholder.jpeg'
 import { Button } from '../../../common/components/Buttons'
-import { colors } from '../../../common/theme/colors'
 import * as ROUTES from '../../../common/utils/constants/routes'
-import * as Styles from './Welcome.styles'
+import styles from './Welcome.css'
```

Comment:
Please rename Welcome.css to Welcome.module.css

Reply:
renamed